worker/worker/handler.py

```diff
@@ -76,7 +80,8 @@ def process(sess: Session, message: SQSMessageRecord, nonce: int = None) -> Tupl
 
     unsigned_tx = gql.create_action(
         "unload_from_garage", pubkey=account.pubkey, nonce=nonce,
-        fav_data=fav_data, avatar_addr=message.body.get("avatar_addr"), item_data=item_data,
+        fav_data=fav_data, avatar_addr=avatar_address, item_data=item_data,
+        memo=f"agent_address: {agent_address}, avatar_address: {avatar_address}"
```

Comment:
It seems to be a little bit hard to parse. how about using simple json notation as like below...

```python
memo=json.dumps([agent_address, avatar_address])
```